weather_app/weather/views.py

```diff
@@ -1,9 +1,32 @@
+from multiprocessing import context
+import random
+import time
+from tkinter.font import names
+from unicodedata import name
+from django.http import HttpResponse
 from django.shortcuts import render, redirect
 from django.conf import settings
 import requests
 from .models import City
 from .forms import CityForm
 # Create your views here.
+def index_prueba(request):
+    name = [
```

Comment:
```suggestion
    names = [
```
El problema es que la variable la tenias como ```name``` pero mas  adelante esta como ```names```.
Por convencion, como es una lista, dejemos el nombre en plural como ```names``` y con eso en la lineas 19 donde se utiliza la variable ya queda funcionando